plugins/hls-retrie-plugin/src/Ide/Plugin/Retrie.hs

```diff
@@ -267,13 +276,13 @@ describeRestriction restrictToOriginatingFile =
         if restrictToOriginatingFile then " in current file" else ""
 
 suggestTypeRewrites ::
-  (Outputable (IdP pass)) =>
+  (Outputable (IdP GhcRn)) =>
```

Comment:
I need to check again whether `GhcRn` is required. It was needed in a prior iteration.

Reply:
It is